src/shared/schema/avl.schema.ts

```diff
@@ -91,6 +92,7 @@ export type SiriVehicleActivity = z.infer<typeof vehicleActivitySchema>;
 export const siriSchema = z.object({
     ServiceDelivery: z.object({
         ResponseTimestamp: z.string(),
+        ItemIdentifier: z.string().uuid().optional(),
```

Comment:
Wouldn't it be better to omit the `uuid()` check? If the ItemIdentifier is set by the producer, it can be a format other than UUID